<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomConfig</class>
 <widget class="QWidget" name="CustomConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CustomConfig</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QtTreePropertyBrowser" name="mConfig" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="mCheckOriginal">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 10, 34);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="mSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>保存设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QClickLabel" name="mClickLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 10, 34);</string>
         </property>
         <property name="text">
          <string>显示更多功能：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mIndicate">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 10, 34);</string>
         </property>
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="mCalculate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(35, 6, 255);</string>
       </property>
       <property name="text">
        <string>立即计算</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">QtTreePropertyBrowser</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QClickLabel</class>
   <extends>QLabel</extends>
   <header>QClickLabel.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mSave</sender>
   <signal>clicked()</signal>
   <receiver>CustomConfig</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancel</sender>
   <signal>clicked()</signal>
   <receiver>CustomConfig</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>609</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mClickLabel</sender>
   <signal>clicked()</signal>
   <receiver>CustomConfig</receiver>
   <slot>onShowMore()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>622</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCalculate</sender>
   <signal>clicked()</signal>
   <receiver>CustomConfig</receiver>
   <slot>onCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>678</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>754</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSave()</slot>
  <slot>onCancel()</slot>
  <slot>onShowMore()</slot>
  <slot>onCalculate()</slot>
  <slot>onStatistic()</slot>
 </slots>
</ui>
